src/Symfony/Bundle/TwigBundle/Resources/config/twig.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Bridge\Twig\AppVariable;
+use Symfony\Bridge\Twig\DataCollector\TwigDataCollector;
+use Symfony\Bridge\Twig\ErrorRenderer\TwigErrorRenderer;
+use Symfony\Bridge\Twig\Extension\AssetExtension;
+use Symfony\Bridge\Twig\Extension\CodeExtension;
+use Symfony\Bridge\Twig\Extension\ExpressionExtension;
+use Symfony\Bridge\Twig\Extension\HttpFoundationExtension;
+use Symfony\Bridge\Twig\Extension\HttpKernelExtension;
+use Symfony\Bridge\Twig\Extension\HttpKernelRuntime;
+use Symfony\Bridge\Twig\Extension\ProfilerExtension;
+use Symfony\Bridge\Twig\Extension\RoutingExtension;
+use Symfony\Bridge\Twig\Extension\StopwatchExtension;
+use Symfony\Bridge\Twig\Extension\TranslationExtension;
+use Symfony\Bridge\Twig\Extension\WebLinkExtension;
+use Symfony\Bridge\Twig\Extension\WorkflowExtension;
+use Symfony\Bridge\Twig\Extension\YamlExtension;
+use Symfony\Bridge\Twig\Translation\TwigExtractor;
+use Symfony\Bundle\TwigBundle\CacheWarmer\TemplateCacheWarmer;
+use Symfony\Bundle\TwigBundle\DependencyInjection\Configurator\EnvironmentConfigurator;
+use Symfony\Bundle\TwigBundle\TemplateIterator;
+use Twig\Environment;
+use Twig\Extension\DebugExtension;
+use Twig\Loader\ChainLoader;
+use Twig\Loader\FilesystemLoader;
+use Twig\Profiler\Profile;
+use Twig\RuntimeLoader\ContainerRuntimeLoader;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+
+    ->set('twig', Environment::class)
+    ->public()
+    ->args([ref('twig.loader'), abstract_arg('Twig options')])
+    ->call('addGlobal', ['app', ref('twig.app_variable')])
+    ->call('addRuntimeLoader', [ref('twig.runtime_loader')])
+    ->configurator([ref('twig.configurator.environment'), 'configure'])
+    ->tag('container.preload', ['class' => 'Twig\Cache\FilesystemCache'])
```

Comment:
Any specific reason why not using the `::class` magic constant here?

Reply:
fixed everywhere